src/preload.js

```diff
@@ -22,6 +28,26 @@ function init() {
   window.MakerBridge = {
     getVersion,
   };
+  window.flashBoardFirmware = flashBoardFirmware;
```

Comment:
It might make sense to put this on the `MakerBridge` object and put as little as possible into the global namespace.  `SerialPort` is not on `MakerBridge` because some of the libraries we're using (that expect to be running on node) assume it will be in the global namespace.